graphql/schema.graphql

```diff
@@ -24,14 +25,16 @@ type Query {
     sortedBy: String = "name"
   ): OrganizationResult
     @cacheControl(scope: PRIVATE, maxAge: SHORT)
-    @validateStoreUserAccess
+    @validateAdminUserAccess(orgPermission: "buyer_organization_view")
+    @validateStoreUserAccess(orgPermission: "buyer_organization_view")
```

Comment:
We should have just one validate directive per API (in this case we should keep only the `@validateStoreUserAccess`